Shift browse props

BboyAkers opened the pull request.

dawsontoth commented:
In https://github.com/HarperDB/hdbms/pull/543 I refactored the names of `databases` and `tables`, which will cause conflicts here, sorry!

dawsontoth committed:
Refetch describe all for clusters properly

https://harperdb.atlassian.net/browse/STUDIO-320
https://harperdb.atlassian.net/browse/STUDIO-322
